recipe/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "oblivious" %}
+{% set version = "7.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/oblivious-{{ version }}.tar.gz
+  sha256: 0f9a0517e059c9a5f3a54c1313761603bbfa34d5aa15b1a258437d4833d24b60
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: True  # [py<38]
+
+requirements:
+  host:
+    - python
+    - libsodium 1.0.18
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python
+    - libsodium >=1.0.18,<1.0.19.0a0
+    - parts >=1.6
+    - fe25519 >=1.4
+    - ge25519 >=1.4
+    - bn254 >=0.1
```

Should we set an upper bound?
```suggestion
    - parts >=1.6,<2
    - fe25519 >=1.4,<2
    - ge25519 >=1.4,<2
    - bn254 >=0.1,<0.2
```